index.js

```diff
@@ -0,0 +1,79 @@
+var Eris = require('eris')
+
+let pingCount = 0
+const newsChannel = process.env.NEWS_CHANNEL
+
+const bot = new Eris.CommandClient(process.env.TOKEN, {}, {
+    description: "A helpful server bot",
+    owner: "Errolyn",
+    prefix: "!"
+});
+
+bot.on("ready", () => {
+    console.log("Ready!");  
+});
+
+bot.on("error", err => {
+    console.log(err);
+});
+
+bot.registerCommand(
+    "ping", 
+    () => {
+        incPing()
+        return("I have been pinged " + pingCount + " times since I last took a nap.")
+    },
+    {
+        description: "Pong!",
+        fullDescription: "Used to see if the bot is responding."
+    }
+);
+
+bot.registerCommand(
+    "news", 
+    (msg) => {
+        const userName = msg.author.username;
+        const messageChannelName = msg.channel.name;
+        const contentForNewsChannel = stripContent(msg.content);
+
+
+        let attachments = getAttachments(msg.attachments);
+        let content = `${userName} posted in ${messageChannelName}: \n${contentForNewsChannel} \n${attachments}`;
+
+        createMessage(newsChannel, content, attachments);
+    }, 
+    {
+        description: "News feed",
+        fullDescription: "Use this command to add content to #news-feed."
+    }
+)
+
+bot.connect();
+
+// helper functions
+
+function incPing(){
+    pingCount++
+}
+
+function stripContent(messageContent){
+    const stringParts = messageContent.split(' ');
+    stringParts.shift();
+
+    const userPost = stringParts.join(' ');
+    return(userPost);
+}
+
+function createMessage(channel, content){
+    bot.createMessage(channel,{
+        content: content,
+    })
+}
+
+function getAttachments(attachments){
+    const attachmentString = []
+    if(attachments !== undefined){
+        attachments.map( (attachment) => { attachmentString.push(`\n ${attachment.url}`) })
```

Comment:
Generally, `map` and `push` is a sign that something isn't quite right. `map` is already building up an array, so `push` is redundant. `map` should always be returning a value. If you don't need or use the return value, you actually want [`Array.forEach`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/forEach).